client/src/redux/profile/profileSlice.ts

```diff
@@ -0,0 +1,34 @@
+import {ISlicesNames} from '../types';
+import createGenericSlice from '../createGenericSlice';
+import {IProfileState} from './types';
+import {changeUserName, changeUserPassword} from './thunks';
+
+const reducers = {};
+
+const initialData = {
+    profile: {

```

Comment:
you don't use this values

Reply:
![image](https://github.com/mykhalenych/UseeSvit/assets/90928916/5109d705-3ba8-40bf-8234-8a997666fade)the function expects an argument as a parameter

Reply:
need to discuss on meeting